### ISSUE TITLE
PrependTo doesn't work with DownValues

### ISSUE THREAD

**Issue by vsht** [mathics/Mathics#1251]
As the title says, the following doesn't work

```
DownValues[foo]
DownValues[foo]={x_^2:>y}
PrependTo[DownValues[foo], {x_^3:>z}]
DownValues[foo]={x_^3:>z}
```
One can assign DownValues directly just fine, but not via PrependTo. Mma accepts this code without any issues

```
In[1]:= DownValues[foo]                                                                                                                      

Out[1]= {}

In[2]:= DownValues[foo]={x_^2:>y}                                                                                                            

             2
Out[2]= {(x_)  :> y}

In[3]:= PrependTo[DownValues[foo], {x_^3:>z}]                                                                                                

              3                        2
Out[3]= {{(x_)  :> z}, HoldPattern[(x_) ] :> y}

In[4]:= DownValues[foo]={x_^3:>z}                                                                                                            

             3
Out[4]= {(x_)  :> z}
```

This issue is relevant for #1174

**Comment by mmatera** [comment]
Should be addressed by #1257, now merged in master.

**Comment by rocky** [comment]
I think this has been addressed and double checked. If it is still lacking, feel free to re-open with comment.
